README.md

```diff
@@ -55,31 +61,38 @@ PR, use the JF_INCLUDE_ALL_VULNERABILITIES environment variable.
 
 #### GitLab
 
-For security reasons, Frogbot is not triggered automatically.
+For security reasons, Frogbot cannot be automatically triggered.
 After you create a new pull request, the maintainer of the git repository can trigger Frogbot to scan the pull request
 from the pull request UI. The scan output will include only new vulnerabilities added by the pull request.
-Vulnerabilities that aren't new, and existed in the code prior to the pull request creation, will not be included in the
+Vulnerabilities that existed in the code prior to the pull request creation, will not be included in the
 report. In order to include all of the vulnerabilities in the report, including older ones that weren't added by this
 PR, use the JF_INCLUDE_ALL_VULNERABILITIES environment variable.
```

Comment:
Shouldn't we add this inside a NOTE as well?

Reply:
You're right. Done.